README.md

```diff
@@ -156,7 +157,7 @@ true|false );
 ### Configuration Options <a name="configOpts"></a>
 + `year`: Retrieve calendar dates for the given year (year should be an integer). Defaults to the current system year if not specified
 + `country`: Include celebration dates requested by the Episcopal council(s) of the given country that have been approved by the Holy See. If not specified, no National dates are included in the calendar output. If an unrecognized country is specified, romcal will silently ignore the property and will not return any National dates in the calendar output. Country names should be specified in camel case (i.e. `unitedStates`, `czechRepublic`).
-+ `locale`: Defaults to `en-US` (english) if not set. Romcal celebration names can be localized to different languages. If a given locale does not have the localized name for a celebration in that language, romcal will fallback to use the celebration name in English.
++ `locale`: Defaults to `en` (english) if not set. Romcal celebration names can be localized to different languages. If a given locale does not have the localized name for a celebration in that language, romcal will fallback to use the celebration name in the base language (if a region was specified in the locale), and finally in English. More details on locales management in the [localisation section](#localisation).
```

Comment:
1. _english_ → _English_; probably even to _US English_, but I leave that to your consideration.

2. _romcal_ → _Romcal_

3. _fallback_ → _fall back_

4. I would omit _section_ word from _localisation section_.

> + `locale`: Defaults to `en` (English) if not set. Romcal celebration names can be localized to different languages. If a given locale does not have the localized name for a celebration in that language, Romcal will fall back to use the celebration name in the base language (if a region was specified in the locale), and finally in English. More details on locales management in the [localisation](#localisation).